tests/ra_aid/tools/test_reflection.py

```
@@ -0,0 +1,62 @@
+    """
+    return True
+
+def complex_func(pos1, pos2, *args, kw1="default", **kwargs):
+    """A function with complex signature."""
+    pass
+
+def no_docstring_func(x):
+    pass
+
+class TestGetFunctionInfo:
+    def test_simple_function_info(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_simple_function_info`

Add return type annotation: `None`

(ANN201)